scripts/ci/verify-build.js

```diff
@@ -195,13 +193,18 @@ function startDevelopmentServer( cwd ) {
 		sampleServer.stdout.on( 'data', data => {
 			const content = stripAnsiEscapeCodes( data.toString() ).slice( 0, -1 );
 			const endMatch = /webpack \d+\.\d+\.\d+ compiled successfully in \d+ ms/.test( content );
+			const errorMatch = content.indexOf( 'ERROR' ) !== -1;
```

Comment:
When there are some errors in `sample/*` files after the server starts there is no information what is going. I've added a naive error handler it helped me few times while debugging.